src/ots.cc

```diff
@@ -702,8 +702,9 @@ bool ProcessGeneric(ots::FontFile *header,
       // We don't sanitize bitmap tables, but don’t reject bitmap-only fonts if
       // we are asked to pass them thru.
       // Also don’t reject if we are asked to pass glyf/loca thru.
-      if (!font->GetTable(OTS_TAG('C','B','D','T')) &&
-          !font->GetTable(OTS_TAG('C','B','L','C'))) {
+      if (!font->GetTable(OTS_TAG('C', 'B', 'D', 'T')) &&
+          !font->GetTable(OTS_TAG('C', 'B', 'L', 'C')) &&
+          !font->GetTable(OTS_TAG('C', 'F', 'F', '2'))) {
```

Comment:
I think this check should be move up next to the `if (font->GetTable(OTS_TAG_CFF))` check.

Reply:
Thanks for the review, sounds reasonable. Updated the patch accordingly.